Maskbook/Scene/Wallet/WalletSend/SendTransactionCofirmPopViewController.swift

```diff
@@ -311,8 +314,15 @@ final class SendTransactionCofirmPopViewController: UIViewController {
                      self.gasFeeEthLabel.text = "\(costEth) \(maskUserDefaults.network.mainToken?.symbol ?? "")"
                      let tokenValue = NSDecimalNumber(string: self.viewModel.selectedTokenPublisher.value?.calculateAmountValue(quantity: self.amount))
                      let gasValue = NSDecimalNumber(string: EthUtil.getGasFee(gwei: gasPriceValue, gasLimit: gasLimitValue, price: token?.price as! Double))
+                     
                      let roundBehavior = NSDecimalNumberHandler(roundingMode: .plain, scale: 2, raiseOnExactness: false, raiseOnOverflow: false, raiseOnUnderflow: false, raiseOnDivideByZero: false)
-                     self.totalPriceLabel.text = "\(maskUserDefaults.currency.symbol)\(tokenValue.adding(gasValue).rounding(accordingToBehavior: roundBehavior).stringValue)"
+                     let totalValue = tokenValue.adding(gasValue).rounding(accordingToBehavior: roundBehavior)
+                     if totalValue.compare(NSDecimalNumber(string: "0.01")) == .orderedAscending{
```

Comment:
Could we use `token.displayBalance` instead here?

Reply:
Here is gas + token.balance